app/views/shipit/ccmenu/project.xml.builder

```diff
@@ -0,0 +1,26 @@
+# Derived from http://timnew.me/blog/2013/04/07/multiple-project-summary-reporting-standard-cctray-xml-feed/
+last_status = 'Unknown'
+if stack.locked?
+  last_status = 'Failure'
+else
+  if deploy.running?
+    last_status = 'pending'
+  elsif deploy.failed?
+    last_status = 'Failure'
+  elsif deploy.error?
+    last_status = 'Exception'
+  elsif deploy.success?
+    last_status = 'Success'
+  end
+end
```

Comment:
Not sure if builder supports it, but things like this usually go into a view helper: http://guides.rubyonrails.org/action_view_overview.html